docs/sources/setup-grafana/configure-security/planning-iam-strategy.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Planning IAM integration strategy
```

Comment:
This topic should be in an index file within its own planning-iam-strategy directory.

Reply:
Moved!